### PR TITLE
[doc] Add guide on how to Set Up SeaTunnel with Kubernetes

### PR DESCRIPTION
**Pull request by GezimSejdiu**
## Purpose of this pull request

Hey SeaTunnel community, 

This pull request provides a guide (as of now, Flink only) on using/setting up SeaTunnel via Kubernetes. 

PS: I will open another thread/feature/discussion on how to move forward withing sea tunnel docker images. I myself prefer decoupling framework/library from base image (e.g. Flink) so that we also make use of existing community is driven docker images as base images, instead of installing all that within seatunnel base image. I will try to provide something (whenever I find some more time to do it  :) ) and then you can also review/comment to that. This guide/solution takes the tar of the current release (but I do prefer that we also package it within docker and push docker images to `ghcr.io` or `docker hub` so that we do not need to rebuild them locally). Let us discuss this in another thread.

## Check list

* [X] Code changed are covered with tests, or it does not need tests for reason: `doc`
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [X] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: docs/en/start/kubernetes.mdx [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+<Tabs
+  groupId="engine-type"
+  defaultValue="flink"
+  values={[
+    {label: 'Flink', value: 'flink'},
+  ]}>
+<TabItem value="flink">
+
+```shell
```
Comment (zhongjiajie) [inline docs/en/start/kubernetes.mdx]: I think syntax `Dockerfile` is support now, am I right?
```suggestion
```Dockerfile
```

File: docs/en/start/kubernetes.mdx [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-0.1.0/
+
+helm install flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator
+```
+
+You may verify your installation via `kubectl`:
+
+```bash
+➜  seatunnel git:(main) ✗ kubectl get pods
```
Comment (zhongjiajie) [inline docs/en/start/kubernetes.mdx]: We should use native bash instead of custom like `zsh` or `fish`
```suggestion
kubectl get pods
```

**Review by zhongjiajie — COMMENTED** [review]
I add some basic review suggestions, but I am not quit familiar with kubernetes, could some review the commands in this PR?

**Review by kezhenxu94 — COMMENTED** [review]
Maybe you need to add an item in https://github.com/apache/incubator-seatunnel/blob/dev/docs/sidebars.js pointing to this new file?

**Comment by zhongjiajie** [comment]
> Maybe you need to add an item in https://github.com/apache/incubator-seatunnel/blob/dev/docs/sidebars.js pointing to this new file?

good catch! we should add it to https://github.com/apache/incubator-seatunnel/blob/dev/docs/sidebars.js#L71

File: docs/en/start/kubernetes.mdx (hunk 1 above)
Comment (GezimSejdiu) [reply docs/en/start/kubernetes.mdx]: Good catch. I just update to using `Dockerfile` syntax. Thanks for pointing that out.

File: docs/en/start/kubernetes.mdx (hunk 2 above)
Comment (GezimSejdiu) [reply docs/en/start/kubernetes.mdx]: Can't agree more. Just update to using `bash` only.

**Comment by GezimSejdiu** [comment]
Hey @zhongjiajie @kezhenxu94 , 

thanks a lot for your feedback and review on this PR.

I made some changes (based on your suggestions).

> > Maybe you need to add an item in https://github.com/apache/incubator-seatunnel/blob/dev/docs/sidebars.js pointing to this new file?
> 
> good catch! we should add it to https://github.com/apache/incubator-seatunnel/blob/dev/docs/sidebars.js#L71

I also did add it to the sidebar.js -- wasn't aware of that since I thought (checked on the file) that it can also be generated automatically.

**Review by kezhenxu94 — APPROVED** [review]
Thanks!

**Comment by CalvinKirs** [comment]
Well done:)

**Comment by zhongjiajie** [comment]
> I also did add it to the sidebar.js -- wasn't aware of that since I thought (checked on the file) that it can also be generated automatically.

It could be auto-generated in some cases,
https://github.com/apache/incubator-seatunnel/blob/dev/docs/sidebars.js#L90-L94
but when we want content in a specific order instead of alphabetically, we should point them manually

**Pull request merged**